packages/react-instantsearch-hooks/src/useInstantSearch.ts

```diff
@@ -1,4 +1,4 @@
-import instantsearch from 'instantsearch.js';
+import instantsearch from 'instantsearch.js/es';
```

Comment:
Just making sure, removing `/es` would fail the `node test/is-a-module.mjs` test with the newer IS.js dependency?

Reply:
will check, but yes